rpa-builder/6.2/modules/ROOT/pages/activity-library.adoc

```diff
@@ -0,0 +1,68 @@
+= Activity Library
+
+The Activity Library enables you to reuse certain activities that repeat during the automation of business processes, a login for example, and avoid creating the same workflow multiple times.
```

Comment:
```suggestion
The Activity Library enables you to reuse activities that repeat during the automation of business processes, such as a login, to avoid creating the same workflow multiple times.
```

Reply:
(Not sure if that edit changes the meaning.)